src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -299,4 +299,145 @@ In such case, an *Association failed* notification will be triggered to users su
+These icons are the following:
+
+image::../../images/Association-authorrizations-icons.png[align=center]
+
+* The first icon indicates that the user has `read` rights over the association(s).
+* The second icon indicates that the user has `write` rights over the association(s).
+* The third icon indicates that the user has `admin` rights over the association(s).
+
+The figure below shows how the user visualize the authorization icon next to the association.
+
+image::../../images/User-association-icon.png[align=center]
+
+To delete an authorization, the admin simply needs to select one from the list and the delete button will be activated.
+It is the button with a "bin" symbol next to the add a new authorization button "+".
+The figure below shows an example.
+
+image::../../images/Delete-association-authorisation.png[align=center]
+
+Once the admin delete the authorization, it will be removed from the list as shown in the figure below.
+
+image::../../images/Association-authorisations-view.png[align=center]
+
+To update an authorization, the admin needs to click on the drop-down list of the access type or the priority level and select a new value.
+The figure below shows an example.
+
+image::../../images/Update-association-authorisation.png[align=center]
+
+Once the selection is made the authorization will be updated as shown in the figure below.
+
+image::../../images/Updated-association-authorisation.png[align=center]
+
+To manage authorization for all associations owned by a specific user, the process is nearly identical.
+The admin or the associations' owner need to click on the dedicated button above the associations view in the Job-Planner calendar associations portal will be activated.
+The button has a shield-like with a user icon, as shown in the figure below.
+
+image::../../images/Association-owner-button.png[align=center]
+
+A dedicated authorization view will open.
+The figure below shows an example.
+
+image::../../images/Association-owner-view.png[align=center]
+
+Each authorization has the same attributes as an association authorization, however, only the association ID is replaced by `Owner`.
+This attribute is used to give access to all associations created by the specified username.
+
+To create a new associations' owner authorization, the admin just needs to click on the "+" button.
+Then the view of creating authorization (as shown in the figure below) will be popped up.
+The admin needs to fill in the authorization attributes (as presented above), then click the "Add authorization" button.
+The figure below shows an example of an authorization which gives the user named `Joseph` `write` access to all of `admin`'s authorizations.
+
+image::../../images/Create-association-owner-autthorisation.png[align=center]
+
+Once the admin has created an authorization, it will appear in the authorizations list in the authorization view, as shown in the figure below.
```

Comment:
admin  -> user

 it will appear in the authorization view, as shown in the figure below: